src/cart/crt_internal.h

```diff
@@ -70,4 +70,10 @@
 
 extern uint32_t crt_swim_rpc_timeout;
 
+#define DBG_ENTRY()					\
+	D_DEBUG(DB_TRACE, ">>>> Entered %s: %d\n", __func__, __LINE__)
```

Comment:
This will print file/line information twice because D_DEBUG already prints that

Reply:
By the end of this project, all DBG_ENTRY and DBG_EXIT will be eliminated or they will be defined to be NULL.  I don't want them to be there at runtime, it will create a timing hit.  I would prefer that we keep them in for future debugging purpose.  For the time being, they are there so I can see the trace of the calling stack in the log files.